pages/common/zapier-convert.md

```diff
@@ -0,0 +1,16 @@
+# zapier convert
+
+> Convert a Visual Builder integration to a CLI integration.
+> More information: <https://github.com/zapier/zapier-platform/blob/main/packages/cli/docs/cli.md#convert>.
```

Comment:
```suggestion
> More information: <https://platform.zapier.com/reference/cli#convert>.
```

Reply:
We can do this but we will need to update the merged pages aswell